android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -416,10 +419,12 @@ private void restoreTransactions(final Result result) {
         CommonKt.restoreTransactions(getOnResult(result));
     }
 
+    @SuppressWarnings("deprecation")
     private void reset(final Result result) {
```

Comment:
should we also mark these methods as deprecated for flutter users? or does that bubble up from hybrid-common already?

Reply:
they're [already marked as deprecated](https://github.com/RevenueCat/purchases-flutter/blob/main/lib/purchases_flutter.dart#L294), this is just our own usage of the deprecated methods in the implementation